certora/confs/ExternalCallsEthereum.conf

```diff
@@ -0,0 +1,18 @@
+{
+  "files": [
+    "src/MorphoTokenEthereum.sol"
+  ],
+  "parametric_contracts": [
+    "MorphoTokenEthereum"
+  ],
+  "solc": "solc-0.8.27",
+  "verify": "MorphoTokenEthereum:certora/specs/ExternalCalls.spec",
+  "rule_sanity": "basic",
+  "server": "production",
+  "packages": [
+    "@openzeppelin/contracts/=lib/openzeppelin-contracts-upgradeable/lib/openzeppelin-contracts/contracts"
+  ],
+  "loop_iter": "3",
+  "optimistic_loop": true,
+  "msg": "Morpho Token Ethereum Reentrancy"
```

Comment:
```suggestion
  "msg": "Morpho Token Ethereum External Calls"
```

Reply:
Fixed in https://github.com/morpho-org/morpho-token-upgradeable/pull/90/commits/b4f049bf904daddac992bf417f403d44039ea170.